source/codegen/metadata/nidmm/enums.py

```diff
@@ -162,38 +163,63 @@
         ]
     },
     'Configuration4022': {
+        'codegen_method': 'public',
         'values': [
             {
-                'name': 'NIDMM_4022_CONFIG_GUARD',
```

Comment:
Names changed to have NIDMM_VAL prefix instead of NIDMM_ prefix... do we know why?